server/conn_stmt.go

```diff
@@ -110,11 +110,13 @@ func (cc *clientConn) handleStmtPrepare(ctx context.Context, sql string) error {
 				return err
 			}
 		}
+		//if err := cc.flush(ctx); err != nil {
+		//	return err
+		//}
```

Comment:
It should be removed?

Reply:
Yes, it will be removed, but this PR is just a draft and far from ready.